modules/cuspatial/src/addon.cpp

```diff
@@ -21,19 +21,20 @@ struct rapidsai_cuspatial : public nv::EnvLocalAddon, public Napi::Addon<rapidsa
   rapidsai_cuspatial(Napi::Env const& env, Napi::Object exports) : nv::EnvLocalAddon(env, exports) {
     DefineAddon(
       exports,
-      {
-        InstanceMethod("init", &rapidsai_cuspatial::InitAddon),
-        InstanceValue("_cpp_exports", _cpp_exports.Value()),
-        InstanceMethod<&rapidsai_cuspatial::create_quadtree>("createQuadtree"),
-        InstanceMethod<&rapidsai_cuspatial::quadtree_bounding_box_intersections>(
-          "findQuadtreeAndBoundingBoxIntersections"),
-        InstanceMethod<&rapidsai_cuspatial::find_points_in_polygons>("findPointsInPolygons"),
-        InstanceMethod<&rapidsai_cuspatial::find_polyline_nearest_to_each_point>(
-          "findPolylineNearestToEachPoint"),
-        InstanceMethod<&rapidsai_cuspatial::compute_polygon_bounding_boxes>(
-          "computePolygonBoundingBoxes"),
-        InstanceMethod<&rapidsai_cuspatial::compute_polyline_bounding_boxes>(
-          "computePolylineBoundingBoxes"),
+      {InstanceMethod("init", &rapidsai_cuspatial::InitAddon),
+       InstanceValue("_cpp_exports", _cpp_exports.Value()),
+       InstanceMethod<&rapidsai_cuspatial::create_quadtree>("createQuadtree"),
+       InstanceMethod<&rapidsai_cuspatial::quadtree_bounding_box_intersections>(
+         "findQuadtreeAndBoundingBoxIntersections"),
+       InstanceMethod<&rapidsai_cuspatial::find_points_in_polygons>("findPointsInPolygons"),
+       InstanceMethod<&rapidsai_cuspatial::find_polyline_nearest_to_each_point>(
+         "findPolylineNearestToEachPoint"),
+       InstanceMethod<&rapidsai_cuspatial::compute_polygon_bounding_boxes>(
+         "computePolygonBoundingBoxes"),
+       InstanceMethod<&rapidsai_cuspatial::compute_polyline_bounding_boxes>(
+         "computePolylineBoundingBoxes"),
+       InstanceMethod<&rapidsai_cuspatial::lonlat_to_cartesian>("lonLatToCartesian")
+
```

Comment:
This seems to have been reformatted with a different version of clang-format than is inside the dev container?

Reply:
I noticed this. In the container I have `clang-format version 12.0.1`, which is formatting it this way.